.github/workflows/deploy.yml

```diff
@@ -14,6 +14,7 @@ on:
         type: choice
         options:
           - review
+          - staging
```

Comment:
I'm not sure we need this. There is a manual deployment workflow which includes staging i.e. deploy_to_environment.yml